extensions/shared/external-media/media-browser/index.js

```diff
@@ -27,6 +27,7 @@ const EmptyResults = memo( () => (
 function MediaBrowser( props ) {
 	const { media, isLoading, pageHandle, className, multiple, setPath, nextPage, onCopy } = props;
 	const [ selected, setSelected ] = useState( [] );
+	const [ focusedItemIndex, setFocusedItemIndex ] = useState( -1 );
```

Comment:
Why do we need a state to determine which item to focus? We only care about the focus when the load more button is clicked.

Reply:
Great catch! 🙌 We indeed don't need to trigger re-render when that value changes. I used the `useRef` to store that value instead (https://github.com/Automattic/jetpack/pull/15868/commits/7f7fc97da9beea31e27b093d422be91984e28f62).